packages/aragon-wrapper/src/index.js

```diff
@@ -873,6 +873,31 @@ export default class Aragon {
       } catch (_) { }
     }
 
+    return this.calculateDirectPath (sender, destination, directTransaction, finalForwarder)
+
+  }
+
+  /**
+   * Calculate the transaction path for a transaction to `destination`
+   * that invokes `directTransaction`.
+   *
+   * @param  {string} sender
+   * @param  {string} destination
+   * @param  {Object} directTransaction
+   * @param  {string} [finalForwarder]
+   * @return {Array<Object>} An array of Ethereum transactions that describe each step in the path
+   */
+  async calculateDirectPath (sender, destination, directTransaction, finalForwarder) {
+    const minGasPrice = this.web3.utils.toWei('20', 'gwei')
```

Comment:
Please check the new logic for dealing with gas estimations and gas prices that was added in #159, as these changes are basically reverting that.